### ISSUE TITLE
Wrong IPv6 address selection.

### ISSUE THREAD

**Issue by naruto522ru** [openwrt/openwrt#11713]
### Describe the bug

The problem is that the ping,traceroute,opkg (update wget return code 4 error) system **sometimes selects** an IPv6 address. Although my ISP does not provide it to me.
The most interesting thing is that if the domain does not have AAAA-record. For example ping will write bad address.

**P.S** Excuse me for the translator, but English is not my native language and I cannot write it fluently.

### OpenWrt version

r20028-43d71ad93e

### OpenWrt target/subtarget

ramips/mt7620

### Device

Asus RT-AC51U

### Image kind

Official downloaded image

### Steps to reproduce

![изображение](https://user-images.githubusercontent.com/28846406/211118184-ab2e0a96-7678-4178-91ed-31dc0a77e6d3.png)
![изображение](https://user-images.githubusercontent.com/28846406/211118318-fc4ef6fa-9e44-45f0-8c10-d157703cf237.png)
![изображение](https://user-images.githubusercontent.com/28846406/211118420-4dfd4930-6988-4967-9583-25faef4eb444.png)
![изображение](https://user-images.githubusercontent.com/28846406/211118480-efd9689d-adc4-4c0d-8852-8ff692e2ae2f.png)

### Terms

- [x] I am reporting an issue for OpenWrt, not an unsupported fork.

**Comment by jow-** [comment]
Please provide the output of `ip -6 route show table all` as well as `ifstatus wan6; ifstatus wan_6; ifstatus lan`

**Comment by naruto522ru** [comment]
> Please provide the output of `ip -6 route show table all` as well as `ifstatus wan6; ifstatus wan_6; ifstatus lan`

![Screenshot_20230107-033313_Termius](https://user-images.githubusercontent.com/28846406/211122148-85766efd-bc5d-4ae6-b725-c48a9bf06a1f.png)
![Screenshot_20230107-034023_Termius](https://user-images.githubusercontent.com/28846406/211122641-6ddec010-3b07-41d4-ab3f-c680649a7490.png)

But the first two commands did not. I have to put my interface name, but I don't know what my interface name is.

P.S Sorry, but I am new to OpenWRT specifically. That's why I don't know a lot of things.

**Comment by naruto522ru** [comment]
There is an important observation. I also noticed especially it is visible from the phone from the browser. That clients are also given an IPv6 address.
As I have established first by the above-mentioned feature.
And now I decided to connect the second router (by wire) to the above mentioned.
That is, the connection scheme is as follows
_WAN->Router1 (192.168.0.1)->Router2(192.168.10.1)-PC\Phone e.g_
Connecting to the second router via ssh
I enter
`nslookup google.com 192.168.0.1`
![изображение](https://user-images.githubusercontent.com/28846406/211157712-95fbebfe-5f61-473f-b90a-295a32818610.png)
